Android/app/src/main/java/uk/ac/lshtm/keppel/android/scanning/ScanActivity.kt

```diff
@@ -55,9 +55,13 @@ class ScanActivity : AppCompatActivity() {
                 }
 
                 ScannerState.Connected -> {
-                    binding.connectProgressBar.visibility = View.GONE
-                    binding.captureButton.visibility = View.VISIBLE
-                    binding.captureProgressBar.visibility = View.GONE
+                    if (intent.extras?.containsKey(OdkExternal.PARAM_FAST) == true) {
```

Comment:
Should this check the passed value, too? Or does this do that already and I just don't understand Kotlin? 😅 

I.e., as a former designer, I would expect that if I changed:

```
uk.ac.lshtm.keppel.android.fast="true"
```

to

```
uk.ac.lshtm.keppel.android.fast="false"
```

that it would disable fast mode for that question.

Reply:
> that it would disable fast mode for that question.

Good shout! I'll make that happen. That also calls into question what to do for arbitrary values (like "blah"). I'll go with the same semantics as `boolean-from-string` (anything that isn't "1" or "true" is `false`) and that way it should work as expected if the value is a reference (to a calculation for example) rather than a literal.